Felienne/hedy
# [UI IMPROVEMENT] Reduces width of code editor gutter

**Description**
In this PR we decrease the width of the code editor gutter. This had unnecessary margin on the left and the right, especially with more than 10 lines. We set a fixed value of `32px` in contrast to the original `48px`. The resulting issue is that the gutter no longer resizes. As long as a program is shorter than 1000 lines this is no problem, when the program is longer the line counter will go out-of-bound. I expect this not to be an issue.

**Fix for**
This PR fixes #1123.

**How to test**
Navigate to any code page and verify that the gutter is smaller than before. Also notice that it no longer resizes on different line counts.

**Screenshots**
![Schermafbeelding 2022-01-24 om 15 20 43](https://user-images.githubusercontent.com/38155424/150800650-45933b69-7ef6-4351-a877-6b65728e770c.png)
Old view

![Schermafbeelding 2022-01-24 om 15 21 17](https://user-images.githubusercontent.com/38155424/150800658-002f07ff-9a75-4aac-af54-db96362ac7c7.png)
New view
